src/main/java/net/fuzzycraft/botanichorizons/patches/GregtechPatches.java

```diff
@@ -228,33 +276,13 @@ private static IRecipe addSlabRecipe(ItemStack output, ItemStack input, int circ
             ItemStack circuitStack = GT_Utility.getIntegratedCircuit(circuit);
             inputs = new ItemStack[]{input, circuitStack};
         }
-        RecipeMaps.cutterRecipes.addRecipe(
-                true,
-                inputs,
-                new ItemStack[]{output},
-                null, null,
-                new FluidStack[]{Materials.Water.getFluid(4)},
-                null,
-                ticks, volt, 0
-        );
-        RecipeMaps.cutterRecipes.addRecipe(
-                true,
-                inputs,
-                new ItemStack[]{output},
-                null, null,
-                new FluidStack[]{GT_ModHandler.getDistilledWater(3)},
-                null,
-                ticks, volt, 0
-        );
-        RecipeMaps.cutterRecipes.addRecipe(
-                true,
-                inputs,
-                new ItemStack[]{output},
-                null, null,
-                new FluidStack[]{Materials.Lubricant.getFluid(1)},
-                null,
-                ticks / 2, volt, 0
-        );
+        GT_Values.RA.stdBuilder()
+                        .itemInputs(inputs)
+                        .itemOutputs(output)
+                        .duration(ticks)
+                        .eut(volt)
+                        .addTo(cutterRecipes);
+
```

Comment:
Fluid requirements and speed differences have been removed, please restore them.

Reply:
they aren't removed, they are handled in the recipe map, unless you saw them missing in game

Reply:
> `Explicit > Implicit` You no longer need to go look for things buried in multiple helper functions and hidden behind complicated logic.

:eyes:

Reply:
there are a few cases of weird recipes handling, like Implosion compressor, primitive blast furnace, or cutting machine. but those are made like this to ensure no one forgets a recipe when adding it manually. while keeping the usual recipe logic.